pkg/internal/ebpf/httpfltr/httpfltr.go

```diff
@@ -258,7 +251,7 @@ func (p *Tracer) UProbes() map[string]map[string]ebpfcommon.FunctionPrograms {
 }
 
 func (p *Tracer) SocketFilters() []*ebpf.Program {
-	return []*ebpf.Program{p.bpfObjects.SocketHttpFilter}
+	return nil
```

Comment:
I wonder if we can now remove the `SocketFilter` method from the `ebpf.Tracer` interface and the logic that invokes it.

Reply:
Yeah good point!